repo/config/init.go

```diff
@@ -44,6 +44,12 @@ func Init(out io.Writer, nBitsForKeypair int) (*Config, error) {
 			API:     "/ip4/127.0.0.1/tcp/5001",
 			Gateway: "/ip4/127.0.0.1/tcp/8080",
 		},
+		API: API{
+			HTTPHeaders: HTTPHeaders{
+				"Access-Control-Allow-Headers":  "X-Stream-Output, X-Chunked-Output",
+				"Access-Control-Expose-Headers": "X-Stream-Output, X-Chunked-Output",
+			},
+		},
```

Comment:
add them also to `Gateway`, because gateway also has an API. (we need to rename these... not sure on good names)